changelogs/fragments/56420-Fix-win_hostname-for-non-netbios-names.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+- Fix non netbios compliant name handling ()
```

Comment:
Probably good to include the module and issue it is referencing, e.g. `win_hostname`.